src/main/java/org/sonarsource/sonarlint/ls/SonarLintExtendedLanguageClient.java

```diff
@@ -877,4 +877,10 @@ public int hashCode() {
 
   @JsonNotification("sonarlint/submitNewCodeDefinition")
   void submitNewCodeDefinition(SubmitNewCodeDefinitionParams params);
+
+  record InvalidTokenNotificationParams(String connectionId) {
+  }
+
+  @JsonNotification("sonarlint/invalidTokenNotification")
+  void invalidTokenNotification(InvalidTokenNotificationParams params);
```

Comment:
Minor - normally we name notifications to denote action, or a state. Like `verbNoun` (such as `suggestBinding`). So maybe we could call this `sonarlint/invalidToken`, or `sonarlint/notifyInvalidToken`